tools/functions

```diff
@@ -92,8 +84,7 @@ get_user_confirmation() {
     #
     msg=$([ -n "${1:-}" ] && echo -n "$1" || echo -n "Continue? (y/n) ")
     yes=$([ -n "${2:-}" ] && echo -n "$2" || echo -n "y")
-    say_noln "$msg"
-    read c && [ "$c" = "$yes" ] && return 0
+    read -p "$msg" c && [ "$c" = "$yes" ] && return 0
```

Comment:
With this, the message won't have the `[Firecracker devtool]` decorations anymore, right?

Reply:
I don't think its important enough to warrant another revision, just wanted to call it out to ask if it was intentional :)